titles/installation-rhdh/master.adoc

```diff
@@ -4,3 +4,16 @@ include::artifacts/attributes.adoc[]
 :context: installation-rhdh
 :doctype: book
 :imagesdir: images
+
+// ocp deployment
+include::assemblies/assembly-install-rhdh-ocp.adoc[leveloffset=+1]
+
+// aws eks deployment
+include::assemblies/assembly-install-rhdh-eks.adoc[leveloffset=+1]
+
+// aks deployment
+include::modules/installation/proc-rhdh-deploy-aks.adoc[leveloffset=+1]
```

Comment:
Have we finalized our decision to use master.adoc instead of the individual title files? Or am I missing anything?

Reply:
We have not finalized any decisions about the use of master.adoc going forward. We did agree not to remove the workaround from existing titles, but applying the workaround to future titles is moot. It doesn't hurt either way, nor to have a mix of both approaches, but upcoming tooling changes might remove the use of the master.adoc file altogether, making both approaches futile.